app/models/challenge/bylder.rb

```diff
@@ -37,13 +37,15 @@ def associated_user_for_normal_challenge(spot)
         spot.user
       end
 
-      def associated_user_for_tri_challenge(challenger)
-        User.includes(:spot).where(
-          'instrument = ? and part = ? and buck_id != ?',
+      def associated_users_for_tri_challenge(challenger, spot)
+        regular_user = spot.user
+        other_alternate = User.joins('LEFT OUTER JOIN spots on spots.id = users.spot_id').find_by(
```

Comment:
If you want a little less SQL you can do something like this:

```ruby
User
  .left_outer_joins(:spot)
  .where(instrument: challenger.instrument, part: challenger.part)
  .where('file > ?', 12)
  .first
```

Reply:
But I ❤️ sql